ideascube/library/views.py

```diff
@@ -134,7 +134,8 @@ class SpecimenCreate(CreateView):
     def get_initial(self):
         book = get_object_or_404(Book, pk=self.kwargs['book_pk'])
 
-        return {'book': book, 'location': get_server_name()}
+        return {
+            'book': book, 'location': Configuration('server', 'site-name')}
```

Comment:
And the class in the catalog API is called `Package`, not `Pkg`. ;)

I saif I dislike abbreviations for **API names**, I didn't say anything about abbreviations for variables, for example.

That means I would have absolutely no problem writing code like the following:

```python
config = Configuration.get('server', 'site-name')
cfg = Configuration.get('server', 'site-name')
c = Configuration.get('server', 'site-name')
```

However, I think abbreviations make for very poor API names.

--- 

So, in this specific instance, if the line break bothers you then I would fix it with an intermediary variable:

```python
        site_name = Configuration('server', 'site-name')
        return {'book': book, 'location': site_name}
```

Because I consider it a lesser evil than a poor API.